src/ekka.appup.src

```diff
@@ -1,7 +1,10 @@
 %% -*- mode: erlang -*-
 %% Unless you know what you are doing, DO NOT edit manually!!
 {VSN,
-  [{"0.8.1.11",
+  [{"0.8.1.12",
```

Comment:
Just a note: there seems to already be a 0.8.1.13 tag: https://github.com/emqx/ekka/releases/tag/0.8.1.13

Reply:
True. This line denotes "from" version I believe. And 0.8.1.13 is a bit of a mess: was tagged out-of-branch, isn't used anywhere AFAICS, and has no update instructions anymore.

Reply:
True, I see now that this tag apparently removed the appup itself :see_no_evil: